examples/dt/uni07eta/control-plane/service-values.yaml

```diff
@@ -0,0 +1,148 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderVolumes:
+    ontap-iscsi:
+      replicas: 1
+      customServiceConfig: |
+        [ontap]
+        volume_backend_name=ontap
+        volume_driver=cinder.volume.drivers.netapp.common.NetAppDriver
+        netapp_storage_protocol=iscsi
+        netapp_storage_family=ontap_cluster
+        consistencygroup_support = True
+      customServiceConfigSecrets:
+        # TODO(arxcruz): This need to be in another place
+        - cinder-volume-ontap-secrets
+
+  cinderBackup:
+    customServiceConfig: |
+      [DEFAULT]
+      backup_driver = cinder.backup.drivers.swift.SwiftBackupDriver
+    replicas: 1
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:cinder
+
+      [glance_store]
+      default_backend = default_backend
+
+      [default_backend]
+      rootwrap_config = /etc/glance/rootwrap.conf
+      description = Default cinder backend
+      cinder_store_auth_address = {{ .KeystoneInternalURL }}
+      cinder_store_user_name = {{ .ServiceUser }}
+      cinder_store_password = {{ .ServicePassword }}
+      cinder_store_project_name = service
+      cinder_catalog_info = volumev3::internalURL
+      cinder_use_multipath = true
+    default:
+      replicas: 1
+      type: split
+
+  ironic:
+    enabled: true
+
+  swift:
+    enabled: true
+
+  octavia:
+    enabled: true
+
+  heat:
+    enabled: true
+
+  telemetry:
+    enabled: false
+    metricStorage:
+      enabled: true
+    autoscaling:
+      enabled: true
+    ceilometer:
+      enabled: true
+
+  ovn:
+    ovnController:
+      availability-zones:
+        - zone-1
+
+  nova:
+    schedulerServiceTemplate:
+      customServiceConfig: |
+        [filter_scheduler]
+        enabled_filters = AggregateInstanceExtraSpecsFilter, AvailabilityZoneFilter,ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,DifferentHostFilter,SameHostFilter,AllHostsFilter,ServerGroupAffinityFilter,ServerGroupAntiAffinityFilter
+
+  neutron:
```

Comment:
For Uni07eta job, it should be the neutron config of vlan_provider.yaml: 
https://github.com/openstack-k8s-operators/architecture/pull/73/files#diff-2c13260a55330f998d004019e04c0a74928394530e1cf409bf775aff35049412

Reply:
Hi, 
The Uni07eta job, needs to be updated with neutron config with the link above(vlan_provider) again.
It seems it has alpha config.

Reply:
@dsariel Would you update the Neutron config here or as a separate change?